### PR TITLE
merge brain meningioma

### LINKED ISSUE #7138: MONDO:0850302 & MONDO:0000642 Intracranial/brain meningioma[Merge]
**Mondo term (ID and Label)**
MONDO:0000642 brain meningioma

**Reason for deprecation**
C0349604 from UMLS and 302820008 from SNOMEDCT and a few other sources in UMLS treat intracranial meningioma and meningioma of brain as synonyms.

Not sure if that is correct or if there is a clinically/biologically relevant distinction... the patient-focused portion of the American Association of Neurological Surgeons equates "brain tumors" with "intracranial tumors"  https://www.aans.org/en/Patients/Neurosurgical-Conditions-and-Treatments/Brain-Tumors 

**Term to be merged with**
MONDO:0850302 intracranial meningioma

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)
I

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7138

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -522293,8 +522283,15 @@ name: intracranial meningioma
...
-xref: DOID:0080842 {source="MONDO:equivalentTo"}
-is_a: MONDO:0016642 {source="DOID:0080842"} ! meningioma
+synonym: "brain meningioma" EXACT [MONDO:0000642]
+synonym: "brain meningioma (disease)" EXACT [MONDO:patterns/location]
+synonym: "meningioma (disease) of brain" EXACT []
+xref: DOID:0060106 {source="MONDO:equivalentTo"}
+xref: DOID:0080842 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
+is_a: MONDO:0016642 {source="DOID:0060106", source="DOID:0080842", source="MONDO:Redundant"} ! meningioma
+intersection_of: MONDO:0016642 ! meningioma
+intersection_of: disease_has_location UBERON:0000955 ! brain
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:522293]: @nicolevasilevsky Is this correct? I think it would be more correct to have the location in the meninges.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky I am questioning whether "location in brain" is correct. I set this PR as "request changes". 
The rest looks good.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:522293]: this came from the obsoleted term. I agree this is wrong:
https://www.mayoclinic.org/diseases-conditions/meningioma/symptoms-causes/syc-20355643

I will delete it

**Review by sabrinatoro — APPROVED** [review]
looks good! Thank you!

**Pull request merged**